src/repository/ParticipantRepository.spec.ts

```diff
@@ -0,0 +1,89 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import DatabaseClient from "../database";
+import ParticipantRepository from './ParticipantRepository';
+import { Participant } from '../types/participant';
+
+const mongod = new MongoMemoryServer();
+
+async function asyncForEach(array, callback) {
+  // eslint-disable-next-line no-plusplus
+  for (let index = 0; index < array.length; index++) {
+    // eslint-disable-next-line no-await-in-loop
+    await callback(array[index], index, array);
+  }
+}
+
+describe("Participant Repository", () => {
+  let client: DatabaseClient;
+
+  beforeAll(async () => {
+    const url = await mongod.getUri();
+    const database = await mongod.getDbName();
+
+    client = new DatabaseClient({
+      url,
+      database,
+    });
+    await client.connect();
+
+  });
+
+  beforeEach(async () => {
+    await client.getCollection("participants").deleteMany({}) // Clean the collection in the test database
+  })
+
+  afterAll(async () => {
+    await client.close(true);
+    await mongod.stop();
+  });
+  
+  it("should init the repository",async () => {
+    ParticipantRepository.init(client)
+    expect(ParticipantRepository.participantCollection).not.toBeUndefined()
+  })
+
+  it('should increment participant points by 1', async () => {
+    await ParticipantRepository.participantCollection.insertOne({name:"LucasMontano", points: 1})
+
+    await ParticipantRepository.incrementParticipantPoints("LucasMontano")
+
+    const LucasMontanospoints = (await ParticipantRepository.participantCollection.findOne({name: "LucasMontano"})).points
+    expect(LucasMontanospoints).toBe(2);
+  });
+
+  it('should fetch the top 10', async () => {
+    const top10 = [
```

Comment:
You could add a type to the array, example:
```js
const top10: Array<string>= [...]
```

Reply:
It's only used in local scope. So it doesn't need to be typed. And also it's a test.